configure

```diff
@@ -3,36 +3,16 @@
 ## This micro-configure shell script is here only to
 ## launch the real configuration via ocaml
 
-ocaml=ocaml
-script=./configure.ml
-
-if [ ! -f $script ]; then
-    echo "Error: file $script not found in the current directory."
-    echo "Please run the configure script from the root of the coq sources."
-    echo "Configuration script failed!"
-    exit 1
+dune=dune
```

Comment:
What's the point of this?

Reply:
No particular point other than using a variable as the command is referenced twice.

Reply:
IMO we shouldn't have this variable